Makefile

```diff
@@ -1062,15 +1070,12 @@ endif
 # Packaging components
 #
 ##############################
-
-.PHONY: package
-package:
-	$(V1) cd $@ && $(MAKE) --no-print-directory $@
-
-.PHONY: standalone
-standalone:
+PACKAGE_TARGETS = package_ground package_flight package_installer package_matlab package_all standalone package_info
+PACKAGE_TARGETS += package_ground_compress package_flight_compress package_matlab_compress package_all_compress
```

Comment:
Well i mean PACKAGE_TARGETS includes lots of things and then package_all_compress. doesn't package_all_compress include all those things?